.github/json_matrices/build-matrix.json

```diff
@@ -21,7 +21,7 @@
     {
         "OS": "macos",
         "NAMED_OS": "darwin",
-        "RUNNER": "macos-latest",
+        "RUNNER": "macos-14",
```

Comment:
Are they both use arm CPU?

Reply:
Yep, it's the same, we want to pin the version.
We will create a self-hosted of newer mac on x86